dev_tools/Dockerfile

```diff
@@ -11,14 +21,24 @@ ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8 
 
 # Make python3 default
-RUN rm -f /usr/bin/python && ln -s /usr/bin/python3 /usr/bin/python
+RUN rm -f /usr/bin/python \
+     && ln -s /usr/bin/python3 /usr/bin/python
 
 # Get a current copy of Cirq from Github.
 RUN git clone https://github.com/quantumlib/Cirq
 
+COPY requirements.txt .
+COPY Cirq/cirq/contrib/contrib-requirements.txt .
+COPY Cirq/dev_tools/conf/pip-list-dev-tools.txt .
+
+RUN cat requirements.txt contrib-requirements.txt pip-list-dev-tools.txt >> dependencies.txt \
+     && pip3 install --user -r dependencies
+
 # Install the needed Python libraries.
-RUN pip3 install -r Cirq/requirements.txt -r Cirq/cirq/contrib/contrib-requirements.txt -r Cirq/dev_tools/conf/pip-list-dev-tools.txt
+RUN pip3 install --user .
```

Comment:
I'm surprised that this works - `pip install --user .` would install the current directory, but Cirq is in the `./Cirq` directory...

Reply:
Looks like I made an error here when cleaning before pushing to github. Fixing it rn